torch/nn/modules/normalization.py

```diff
@@ -1,4 +1,46 @@
 from .module import Module
+from .. import functional as F
+
+
+class LocalResponseNorm2d(Module):
```

Comment:
Can we make a generic `LocalResponseNorm`?

Reply:
Made it for 1D/2D/3D. Also fixed space-preserving padding when even-sized neighbourhoods are used (included a test for this).